profile/README.md

```diff
@@ -1,5 +1,5 @@
 ## Hi there 👋
-
+We are Quartz
```

Comment:
This is kind of boring. Can you make it better?

Reply:
Hey I fixed it boss.